aptos-move/aptos-vm-types/src/change_set.rs

```diff
@@ -25,8 +28,8 @@ use std::collections::{
 pub struct VMChangeSet {
     resource_write_set: HashMap<StateKey, WriteOp>,
     module_write_set: HashMap<StateKey, WriteOp>,
-    aggregator_write_set: HashMap<StateKey, WriteOp>,
-    aggregator_delta_set: HashMap<StateKey, DeltaOp>,
+    aggregator_write_set: HashMap<AggregatorID, WriteOp>,
```

Comment:
I would keep StateKeys here, because anyway we are putting in work to create the VMChangeset that represents V1 separately, V1 AggregatorID will be legacy that will wrap statekey, so we can already use StateKey here?